src/sentry/feedback/usecases/create_feedback.py

```diff
@@ -265,55 +291,41 @@ def create_feedback_issue(event, project_id: int, source: FeedbackCreationSource
         },
     )
 
-    if should_filter_feedback(event, project_id, source):
+    project = Project.objects.get_from_cache(id=project_id)
+
+    should_filter, filter_reason = should_filter_feedback(event, project_id, source)
+    if should_filter:
+        track_outcome(
+            org_id=project.organization_id,
+            project_id=project_id,
+            key_id=None,
+            outcome=Outcome.FILTERED,
```

Comment:
The `FILTERED` outcome is reserved for inbound filters and dynamic sampling, I recommend `Outcome.INVALID` here. See for example https://github.com/getsentry/sentry/blob/94a5865f0fac911d2bed5ab2ce950bb0e93fedfd/src/sentry/event_manager.py#L2367-L2368.